algo/uidlist_test.go

```diff
@@ -258,6 +258,30 @@ func TestUIDListIntersect5(t *testing.T) {
 	require.Equal(t, []uint64{3}, u.Uids)
 }
 
+func TestUIDListIntersect6(t *testing.T) {
```

Comment:
does go test framework provide a way to parameterize the values you want to test with (fllNums here). otherwise may be we can write a single test looping through the values ?